src/edu/csus/ecs/pc2/core/imports/LoadICPCTSVData.java

```diff
@@ -165,7 +168,13 @@ public boolean loadFiles(String filename, boolean sendToServer, boolean useConfi
                 info("Load from file " + groupsFilename);
                 info("Load from file " + teamsFilename);
                 
-                info("Added " + accounts.length + " accounts and " + groups.length + " groups.");
+                info("There are now " + accounts.length + " team accounts and " + groups.length + " groups.");
+                
+                if(useConfirmGUI)
+                {
+                    String message = "Updated/Added " + NL + accounts.length + " team accounts and " + NL + groups.length + " groups?";
+                    FrameUtilities.showMessage(null,"TSV Files loaded", message);
```

Comment:
The tense of the new message doesn't quite make sense.  It says "Updated/Added..." (which is past tense and implies that the action has already taken place), whereas the message ENDS with a question mark.  What question is being asked if the action has been done?

Reply:
Yea I saw that.  Will update the messge.

Reply:
Typo, removed the ?